javascript/computer_science/project_binary_search_trees.md

```diff
@@ -66,3 +67,13 @@ Write a simple driver script that does the following:
 1. Print out all elements in level, pre, post, and in order
 
 </div>
+
+### Additional Resources
+
+This section contains helpful links to related content. It isn't required, so consider it supplemental.
+
+If you need more help on the three types of binary search tree traversal algorithms, these video resources may aid you:
+
+- [Inorder Traversal Algorithm](https://www.youtube.com/watch?v=4_UDUj1j1KQ&t=1s) of a binary search tree by Yicheng Gong
```

Comment:
I like the way this is added here, but could you explain why and when a learner should watch these? Something in the lines of "this helps in further visualising how a call stack looks when traversing the tree" should be cool enough IMO.

Reply:
This has been implemented